recipes/oracle-jdk/recipe.yaml

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
Remove this file too.

Reply:
We do not like `boa`?

Reply:
We like boa. We don't use it in feedstocks and so do not need the files.